### ISSUE TITLE
Recruite a Friend (RAF) integration to Registration

### ISSUE THREAD

**Issue by Hellangel1987** [paintballrefjosh/MaNGOSWebV4#58]
I will suggest an integration on the RAF Feature for Trinitycore.

To setup the Recruit a Friend (well manually at least), set the recruiter column in the auth.account table to the id of the account that "recruited" them.

That is, lets say there is an account called Alice (account id 10) and she recruits Bob (account id 20). The following SQL query would do the trick.
Code:
update account set recruiter = 10 where id = 20;

It would be great if this can be integrated in the Registration Form - i will also make a donate via paypal. 

Regards,

Hellangel

**Comment by paintballrefjosh** [comment]
I can look into this, it would be a lot of work to add this.

I imagine this would be done via email, is this what you were thinking?  So Alice would enter Bob's email into a RAF form in MWE and it would generate an email to Bob with a registration URL that links Alice as the recruiter.

Does that make sense?

**Comment by Hellangel1987** [comment]
yes that should fit good !!! would be a plesure if we can realize it and will push this cms

**Comment by sndcode** [comment]
thatd be nice !
